astropy/io/fits/util.py

```diff
@@ -26,8 +25,7 @@
 
 from astropy.utils.exceptions import AstropyUserWarning
 
-from os import PathLike
-path_like = (str, PathLike)
+path_like = (str, os.PathLike)
```

Comment:
according to https://docs.python.org/3/glossary.html#term-path-like-object, path-like should include bytes

Reply:
The three components of python's path-like are string, byte, os.PathLike